test/templates/misc/orphanedArgument.html

```diff
@@ -0,0 +1,6 @@
+{!
+  should ignore and skip rendering orphaned argument
+!}
+<arg orphan>
```

Comment:
that way the unit test is just `<div></div>` and the `<p>` tag is expected to not render